README.md

```diff
@@ -0,0 +1,29 @@
+# Awesome_Books
+
+## Built With
+- HTML5
+- Javascript
+- Linters
+
+
+## Clone This Project
+```
+- $ git clone `$ git clone -b project1 git@github.com:uwadonat/Awesome_Books.git`
+- $ cd Awesome_Books
+```
+
+
```

Comment:
Good job 👍🏽 
Kindly add a link to the live demo of the app to your readme file

Reply:
Thanks for your review
A live demo was added to our readme file